engine/orchestration/src/main/java/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -1460,6 +1469,11 @@ public void orchestrateStart(final String vmUuid, final Map<VirtualMachineProfil
         }
     }
 
+    private boolean checkThatAllVolumesAreAllocated(long vmId) {
```

Comment:
sorry to be a nag (not really) I think 
```suggestion
    private boolean areAllVolumesAllocated(long vmId) {
```
is even better

Reply:
or
```suggestion
    private boolean allVolumesAreAllocatedFor(long vmId) {
```
a boolean method starting with `check...()` seems off a bit. I would expect an exception for those `check..`-methods .